docker-compose.yml

```diff
@@ -1,12 +1,14 @@
 version: '3'
 services:
   postgres:
-    image: postgres:11.5-alpine
+    image: postgres:15.6-alpine
     ports:
       - '127.0.0.1:5432:5432'
     healthcheck:
         test: ["CMD", "pg_isready", "-U", "postgres"]
         interval: 1s
+    environment:
```

Comment:
Needs to be explicit in the new version

Reply:
This results in a scary message in the logs, but [as the docs say](https://www.postgresql.org/docs/14/auth-trust.html):

> trust authentication is appropriate and very convenient for local connections on a single-user workstation.